src/refactor/CSPHandler.py

```diff
@@ -64,7 +64,7 @@ def receive(self, server, port, timeout):
             libcsp.buffer_free(packet)
         return data
 
-    def __uart__(self, device):
+    def _uart(self, device):
```

Comment:
I believe private variables have 2 underscores (https://www.tutorialspoint.com/private-variables-in-python)